includes/src/catalogo.php

```diff
@@ -0,0 +1,40 @@
+
+
+<!DOCTYPE html>
+<html>
+<?php
```

Comment:
Esto debe de estar al comienzo del script. Puede ser un problema porque el session_start() debe generar una cabecera HTTP pero ya habéis empezado a generar salida para el usuario que forma parte del cuerpo de la petición